CvGameCoreDLL/CvGameTextMgr.cpp

```diff
@@ -6492,6 +6492,9 @@ void CvGameTextMgr::parseSpecialistHelpActual(CvWStringBuffer &szHelpString, Spe
 			}
 			bStarted = setResumableCommerceTimes100ChangeHelp(szHelpString, szStart, L": ", L"", aiCommerces, true, bStarted);
 
+			// Happiness
+			bStarted = setResumableValueChangeHelp(szHelpString, szStart, L": ", L"", std::abs(iHappinessChange), iHappinessChange > 0 ? gDLL->getSymbolID(HAPPY_CHAR) : gDLL->getSymbolID(UNHAPPY_CHAR), false, true, bStarted);

```

Comment:
I did a check and there already is such a check in the setResumableValueChangeHelp function itself, so I left it out. Tested it and it works.